server/services/visitService.ts

```diff
@@ -42,15 +42,18 @@ export default class VisitService {
     const token = await this.hmppsAuthClient.getSystemClientToken()
     const orchestrationApiClient = this.orchestrationApiClientFactory(token)
 
-    const contact = bookingJourney.mainContact?.contact
-    const phoneNumber = bookingJourney.mainContact?.phoneNumber
-
-    const visitContact = bookingJourney.mainContact
-      ? {
-          name: typeof contact === 'string' ? contact : `${contact.firstName} ${contact.lastName}`,
-          ...(phoneNumber && { telephone: phoneNumber }),
-        }
-      : undefined
+    const contact = bookingJourney.mainContact
+    const { mainContactEmail, mainContactPhone } = bookingJourney
+
+    let visitContact
+    if (contact) {
+      const name = typeof contact === 'string' ? contact : `${contact.firstName} ${contact.lastName}`
```

Comment:
Just spotted that I've got effectively this same way of getting a contact name in several places: will put in a utils function in next PR.